project-keeper-cli/src/test/java/com/exasol/projectkeeper/cli/ProjectKeeperLauncherIT.java

```diff
@@ -47,20 +47,24 @@ void failsForWrongArguments(final String... args) throws IOException, Interrupte
 
     @Test
     void runMainMethodWithNullArgumentFails() throws IOException, InterruptedException {
-        final IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, ()-> ProjectKeeperLauncher.main(null));
-        assertThat(exception.getMessage(), equalTo("E-PK-CLI-2: Got no or invalid command line argument 'null'. Please only specify arguments 'verify' or 'fix'."));
+        final IllegalArgumentException exception = assertThrows(IllegalArgumentException.class,
+                () -> ProjectKeeperLauncher.main(null));
+        assertThat(exception.getMessage(), equalTo(
+                "E-PK-CLI-2: Got no or invalid command line argument 'null'. Please only specify arguments 'verify' or 'fix'."));
     }
 
     @Test
     void verifyFailsForProjectWithoutGit() throws IOException, InterruptedException {
-        assertProcessFails("verify", "E-PK-CORE-90: Could not find .git directory in project-root " + projectDir
+        final Path projectDirRealPath = this.projectDir.toRealPath();
+        assertProcessFails("verify", "E-PK-CORE-90: Could not find .git directory in project-root " + projectDirRealPath
                 + ". Known mitigations:\n* Run 'git init'.\n* Make sure that you run project-keeper only in the root directory of the git-repository. If you have multiple projects in that directory, define them in the '.project-keeper.yml'.");
     }
 
     @Test
-    void cliDoesNotSupportMavenProjects() throws InterruptedException, IOException {
+    void fixingJavaProjectSucceeds() throws InterruptedException, IOException {
         prepareMavenProject();
-        assertProcessFails("fix", "F-PK-CORE-145: Analyzing Maven projects in standalone mode is not supported. Use project-keeper-maven-plugin for analyzing Maven projects.");
+        assertProcessSucceeds("fix");
+        assertProcessSucceeds("verify");
```

Comment:
This looks like it is actually testing two different things to me, or does a `fix` command spawn both processes? In that case, nevermind.

Reply:
It's a smoke test for the CLI. We first run fix and then verify to check if the fix did fix all findings.
We've proper tests for the individual functionality in the core.